00-introduction.md

```diff
@@ -98,179 +98,180 @@ See [BOLT #11: Invoice Protocol for Lightning Payments](11-payment-encoding.md)
+    release funds. The payment preimage is hashed as the *[payment hash](#Payment-hash)*
+    in the *[HTLC](#HTLC-Hashed-Time-Locked-Contract)*.
+   * _See container: [HTLC](#HTLC-Hashed-Time-Locked-Contract)_
+   * _See derivation: [payment hash](#Payment-hash)_
 
-* *Peers*:
-   * Two *nodes* that are in communication with each other.
+* #### *Peers*:
+   * Two *[nodes](#node)* that are in communication with each other.
       * Two peers may gossip with each other prior to setting up a channel.
-      * Two peers may establish a *channel* through which they transact.
-   * _See related: node_
-
-* *Penalty transaction*:
-   * A transaction that spends all outputs of a *revoked commitment
-    transaction*, using the *commitment revocation private key*. A *peer* uses this
-    if the other peer tries to "cheat" by broadcasting a *revoked
-    commitment transaction*.
-   * _See related: closing transaction, commitment transaction, funding transaction_
-
-* *Per-commitment secret*:
-   * Every *commitment transaction* derives its keys from a per-commitment secret,
+      * Two peers may establish a *[channel](#channel)* through which they transact.
+   * _See related: [node](#node)_
+
+* #### *Penalty transaction*:
+   * A transaction that spends all outputs of a *[revoked commitment
+    transaction](#revoked-commitment-transaction)*, using the *commitment revocation private key*. A *[peer](#peers)* uses this
+    if the other peer tries to "cheat" by broadcasting a *[revoked commitment
+    transaction](#revoked-commitment-transaction)*.
+   * _See related: [closing transaction](#closing-transaction), [commitment transaction](#commitment-transaction), [funding transaction](#funding-transaction)_
+
+* #### *Per-commitment secret*:
+   * Every *[commitment transaction](#commitment-transaction)* derives its keys from a per-commitment secret,
      which is generated such that the series of per-commitment secrets
      for all previous commitments can be stored compactly.
-   * _See container: commitment transaction_
-   * _See derivation: commitment revocation private key_
+   * _See container: [commitment transaction](#commitment-transaction)_
+   * _See derivation: [commitment revocation private key](#commitment-revocation-private-key)_
 
-* *Processing node*:
-   * A *node* that is processing a packet that originated with an *origin node* and that is being sent toward a *final node* in order to route a payment. It acts as a _receiving peer_ to receive the message, then a _sending peer_ to send on the packet.
-   * _See category: node_
-   * _See related: final node, origin node_
+* #### *Processing node*:
+   * A *[node](#node)* that is processing a packet that originated with an *[origin node](#origin-node)* and that is being sent toward a *[final node](#final-node)* in order to route a payment. It acts as a *[receiving peer](#receiving-peer)* to receive the message, then a _sending peer_ to send on the packet.
```

Comment:
nit: you can also add a link for `sending peer`

Reply:
Done ;)